.github/workflows/build_all.yml

```diff
@@ -189,6 +203,33 @@ jobs:
               echo "::set-output name=magma_package::${magma_package}"
             fi
           done
+          for i in `ls -a1 *.deb`
+          do
+            echo "Pushing package $i to JFROG artifiactory: $LF_ARTIFACTORY_BASE_URL"
+            HTTP_RESPONSE=$(curl --silent --write-out "HTTPSTATUS:%{http_code}" -uci-bot:${{ secrets.LF_JFROG_CIBOT_APIKEYS }} -XPUT "$LF_ARTIFACTORY_BASE_URL/magma-packages-test/pool/focal-ci/$i;deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64" -T $i)
+            echo "$HTTP_RESPONSE"
+            # extract the body and download uri
+            HTTP_BODY=$(echo $HTTP_RESPONSE | sed -e 's/HTTPSTATUS\:.*//g')
+            URI=$(echo $HTTP_BODY | jq -r '.uri')
+            if [[ "$URI" != "null" ]]; then
+              ARTIFACTS=$(echo $ARTIFACTS | jq --arg uri $URI '.packages += [$uri]')
+            else
+              PUBLISH_ERROR="true"
+            fi
+            # extract and check status
+            HTTP_STATUS=$(echo $HTTP_RESPONSE | tr -d '\n' | sed -e 's/.*HTTPSTATUS://')
+            if [[ "$HTTP_STATUS" != "2"* ]]; then
+              PUBLISH_ERROR="true"
+            fi
+            # extract magma debian package version
+            match="magma_[0-9]+\.[0-9]+\.[0-9]+-[0-9]+-[a-z0-9]+_[a-z0-9]+.deb"
+            if [[ $i =~ $match ]]; then
+              magma_package=${i#magma_}
+              magma_package=${magma_package%_[a-z0-9]*.deb}
+              magma_package='magma='${magma_package}
+              echo "::set-output name=magma_package::${magma_package}"
+            fi
+          done
```

Comment:
This is a complete code duplication with the only difference in the artifactory URL. :cry: 
This should probably be reduced to a proper abstraction.

Reply:
The plan was just to delete the code block when the license expires. I can rework the code but it was just a way to get it fast